editor/editor_properties.cpp

```diff
@@ -90,6 +90,11 @@ void EditorPropertyText::_text_changed(const String &p_string) {
 		return;
 	}
 
+	// Set tooltip so that the full text is displayed in a tooltip if hovered.
+	// This is useful when using a narrow inspector, as the text can be trimmed otherwise.
+	// Trim to 100 characters to prevent the tooltip from being too long.
+	text->set_tooltip_text(text->get_text().left(TOOLTIP_MAX_LENGTH).strip_edges() + String((text->get_text().length() > TOOLTIP_MAX_LENGTH) ? "..." : ""));
```

Comment:
Might be worth splitting this code into a helper method, and just passing the text to it directly, that avoids calling `get_text` twice as well (also makes sure any edits won't accidentally fail to match the first and second strings)